### PR TITLE
Text

### PR DESCRIPTION
**Pull request by bossmonkey98**
PR review for Buttons

### CODE REVIEW

File: assets/pages/buttons.css [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+.primary-btn{
+    background-color: var(--primary);
+    color: aliceblue;
+    border-radius: 10px;
+}
+.primary-btn:hover, .secondary-btn:hover, .icon-btn:hover, .icon-with-text:hover{
+    background-color: rgb(116, 123, 129);
+}
+.secondary-btn{
```
Comment (bhushannaikaroskar) [inline assets/pages/buttons.css:21]: give spacing here for better readability and better structure

File: assets/pages/buttons.css [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+}
+.icon-btn{
+    background-color: white;
+    color: var(--primary);
+    font-size: large;
+    width: fit-content;
+    height: fit-content;
+    border-radius: 10px;
+}
+.icon-with-text{
```
Comment (bhushannaikaroskar) [inline assets/pages/buttons.css:40]: same goes here I would suggest use spacing wherever required

File: assets/pages/buttons.css [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    align-items: center;
+    justify-content: center;
+    background-color: var(--primary);
+    color: aliceblue;
+    border-radius: 10px;
+}
+.icon-with-text>i{
+    color: aliceblue;
+    margin-right: 0.2em;
+}
```
Comment (bhushannaikaroskar) [inline assets/pages/buttons.css:53]: use font size in px wherever the size is less that 10px.

File: assets/pages/buttons.css [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+    border: 1px solid var(--primary);
+    background-color: white;
+    color: var(--primary);
+    border-radius: 10px;
+}
+.link-btn{
+    text-decoration: underline;
+    color: blue;
+    margin: 0.5rem;
+}
```
Comment (bhushannaikaroskar) [inline assets/pages/buttons.css:31]: Use pixels here

File: assets/pages/buttons.css [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+    margin-top: 3em;
+    height: fit-content;
+    text-align: center;
+}
+.code-snippet>iframe{
+    width: 100%;
+    padding: 0px;
+    margin: 0px;
+    height: 300px;
+}
```
Comment (bhushannaikaroskar) [inline assets/pages/buttons.css:65]: use rem here in height

**Review by bhushannaikaroskar — CHANGES_REQUESTED** [review]
There are multiple places you have used rems instead of px and vice versa. I would say to use pixels wherever its is  less thatn 10px and values in rem wherever it is more than 10px

**Review by bhushannaikaroskar — APPROVED** [review]
As of now I am approving but please do the needful before moving ahead.

File: assets/pages/buttons.css [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+    display: flex;
+    flex-wrap: wrap;
+    align-items: center;
+}
+.primary-btn , .secondary-btn, .icon-btn, .icon-with-text{
+    padding: 1rem;
+    margin: 0.5rem;
+    border: none;
+}
+
```
Comment (zoef-yt) [inline assets/pages/buttons.css:12]: Remove extra spacing.

File: assets/pages/buttons.css [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+    background-color: var(--primary);
+    color: aliceblue;
+    border-radius: 10px;
+}
+.primary-btn:hover, .secondary-btn:hover, .icon-btn:hover, .icon-with-text:hover{
+    background-color: rgb(116, 123, 129);
+}
+.secondary-btn{
+    border: 1px solid var(--primary);
+    background-color: white;
```
Comment (zoef-yt) [inline assets/pages/buttons.css:23]: Use var

**Review by zoef-yt — CHANGES_REQUESTED** [review]
Do make changes suggested by Bhushan

File: assets/pages/buttons.css [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+    color: var(--primary);
+    border-radius: 10px;
+}
+.link-btn{
```
Comment (developer-anand) [inline assets/pages/buttons.css:27]: use px for less than 10

**Review by developer-anand — APPROVED** [review]
Go to go with changes.

**Pull request merged**